src/ui/templates/results/directanswer.hbs

```diff
@@ -1,3 +1,6 @@
+{{#if _config.defaultCard}}
+<div class="yxt-DirectAnswer" data-component="{{_config.defaultCard}}"></div>
```

Comment:
do we want to add a modifier class here to let people conditionally style this class depending on which type of DirectAnswer is used? like `yxt-DirectAnswer yxt-DirectAnswer--something`?

Reply:
sure

Reply:
I think the sdk will automatically add a 'component' css class in this case, but what you said feels more contractual